backend/src/openarchiefbeheer/destruction/api/serializers.py

```diff
@@ -153,6 +153,13 @@ class DestructionListWriteSerializer(serializers.ModelSerializer):
     remove = DestructionListItemWriteSerializer(many=True, required=False)
     reviewer = ReviewerAssigneeSerializer(required=False)
     author = UserSerializer(read_only=True)
+    comment = serializers.CharField(
+        required=False,
+        allow_null=True,
+        help_text=_(
+            "Additional information about the destruction list, such as the reason for creating it."
+        ),
+    )
```

Comment:
```suggestion
```
Since this is a ModelSerializer, you should only need to add it to the Meta `fields`.